packages/chat-component-bindings/src/utils/updateMessagesWithAttached.ts

```diff
@@ -11,36 +11,46 @@ export const updateMessagesWithAttached = (chatMessagesWithStatus: Message[]): v
   chatMessagesWithStatus.sort(compareMessages);
 
   chatMessagesWithStatus.forEach((message, index, messages) => {
-    if (message.messageType !== 'chat') {
-      return;
-    }
-    /**
-     * Attached === true means it is within a group of messages in the current order
-     * Attached === top/bottom means it is on the top/bottom boundary
-     * Attached === false means it is just a single message
-     * A group of messages: continuous messages that belong to the same sender and not intercepted by other senders.
-     */
-    let attached: boolean | MessageAttachedStatus = false;
-    const previousMessage = index > 0 ? messages[index - 1] : undefined;
-    const nextMessage = index === messages.length - 1 ? undefined : messages[index + 1];
+    if (
+      message.messageType === 'chat' ||
```

Comment:
Does this || operator also need to be conditionally compiled out?

Reply:
The in-line @conditional-compile-remove in boolean logic needs to be after the operator || or &&